PhotoPoints/Model/Repository.swift

```diff
@@ -85,6 +100,29 @@ class Repository {
         return nil
     }
     
+    func getImageNameFromXcAssets(item: Item) -> String? {
+        let request = Image.fetchRequest() as NSFetchRequest<Image>
+        let predicate = NSPredicate(format: "item == %@", item)
+        request.predicate = predicate
+        
+        if let image = try? context.fetch(request).first {
+            return image.filename
+        }
+        
+        return nil
+    }
+    
+    func getImageFromFilesystem(item: Item) -> UIImage? {
+        
+        if let firstImage = item.images?.allObjects.first as? Image {
+            let fileName = firstImage.filename
+            print(ImageManager.getPath(for: fileName))
+            return ImageManager.getImage(from: fileName)
+        }
```

Comment:
May need to cast `URL?` to `String` for line 119 to behave (this is presumably a for logging)

Reply:
lets talk about this in the AM

Reply:
It is working as is by the way